### ISSUE TITLE
use laravel Resource Collections instead of transformers

### ISSUE THREAD

**Issue by zohaibhassan156** [yajra/laravel-datatables#1863]
Is there's a way to use Laravel resource Collections instead of transformers

        $data = Sport::select(['id', 'title', 'icon', 'status_id', 'updated_at'])
            ->with('status:id,status')
            ->withCount('users');

        return DataTables::make($data)
            ->setTransformer(new SportTransformer())
            ->toJson();

**Comment by c00p3r** [comment]
if @yajra accepts my [PR](https://github.com/yajra/laravel-datatables-fractal/pull/17) you'll be able to do at least:
```
DataTables::eloquent($query)
->setTransformer(function ($item){
    return Resource::make($item)->resolve();
})
->make();
```

**Comment by c00p3r** [comment]
The feature is released in yajra/laravel-datatables-fractal v1.3.0 
thnx @yajra
